src/core/classes/language-registry.ts

```diff
@@ -18,7 +18,7 @@ export default class LanguageRegistry extends ComponentRegistry<LanguageLike> {
 		if (!this.instances[canonicalId]) {
 			if (this.cache[canonicalId]) {
 				let def = this.cache[canonicalId];
-				let language = new Language(def, this);
+				let language = new Language(def as LanguageProto, this);
```

Comment:
Why is def not already LanguageProto?

Reply:
`def` has type of `LanguageLike`, which is a union of `Language` and `LanguageProto`, so we need to specify it here.